xbmc/pvr/channels/PVRChannelGroup.cpp

```diff
@@ -527,18 +545,18 @@ void CPVRChannelGroup::UpdateClientOrder()
 {
   CSingleLock lock(m_critSection);
 
-  for (const auto& member : GetMembers())
-    member.channel->SetClientOrder(member.iOrder);
+  for (const auto& member : m_sortedMembers)
```

Comment:
You replaced `GetMembers()`by `m_sortedMembers`at several places. I think this is intentional and you double checked that this has no bad side effects?

Reply:
Yes, GetMembers was only called by external classes prior to my last couple of PRs.

Reply:
Okay.